reference/annotation/acl_ancestor.rst

```diff
@@ -0,0 +1,15 @@
+@AclAncestor
+============
+
+This annotation used to protect a controller based on an existing access control list:
+
+.. code-block:: php
+    :linenos:
+
+    /**
+     * @AclAncestor("an_acl_name")
```

Comment:
I think it's better to use `an_acl_id` parameter here to describe to users what here should be an `id` parameter from ACL annotation